site/docs/install-windows.md

```diff
@@ -41,38 +34,63 @@ Alternatively you can:
 *   [Install Bazel from Scoop](#using-scoop)
 *   [Build Bazel from source](install-compile-source.html)
 
-### Step 4 (optional): Configure output directories
+### Step 4: Set up your environment
+
+It's a good idea to rename the Bazel binary to `bazel.exe` and add it to your default paths.
+
+```batch
+set PATH=%PATH%;<path to the Bazel binary>
+```
+
+You can also change your system `PATH` environment variable to make it permanent.
```

Comment:
Can we add a link that explains how to do that?

Reply:
Done